docs/faq.md

```diff
@@ -5,15 +5,27 @@ This page provides help with the most common questions about Amadeus Self-servic
 | Domain                                                                                                                                                          | Questions                                                                             |
 |---------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
 | [Account registration](#account-registration) |  <ul><li>[How do I change my password?](#how-do-i-change-my-password)</li><li>[I registered but never received a confirmation email? What should I do?](#i-registered-but-never-received-a-confirmation-email-what-should-i-do)</li></ul>              |
-| [Self-Service vs Enterprise](#self-service-vs-enterprise) |  <ul><li>[What is the difference between Self-Service and Enterprise APIs?](#what-is-the-difference-between-self-service-and-enterprise-apis)</li><li>[Can I use APIs from both Self-Service and Enterprise?](#can-i-use-apis-from-both-self-service-and-enterprise?)</li></ul>               |
-| [Self-Service APIs](#self-service-apis) |    <ul><li>[Is there a test environment to try the Self-Service APIs?](#is-there-a-test-environment-to-try-the-self-service-apis?)</li><li>[How do I access the Self-Service APIs documentation?](#how-do-i-access-the-self-service-apis-documentation?)</li><li>[Do you provide SDKs?](#do-you-provide-sdks)</li><li>[Where can I see code examples for Amadeus Self-Service APIs?](#where-can-i-see-code-examples-for-amadeus-self-service-apis)</li><li>[How do I make my first Self-Service API call?](#how-do-i-make-my-first-self-service-api-call)</li><li>[How do I move Self-Service APIs from test to production?](#how-do-i-move-self-service-apis-from-test-to-production)</li><li>[How do I delete my application built using Self-Service APIs?](#how-do-i-delete-my-application-built-using-self-service-apis)</li><li>[Will you include more APIs in the Self-Service catalog?](#will-you-include-more-apis-in-the-self-service-catalog?)</li><li>[What are the terms of service for Amadeus Self-Service APIs?](#what-are-the-terms-of-service-for-amadeus-self-service-apis)</li><li>[I am not a travel agent and have no experience in the travel industry, can I still use the Self-Service APIs?](#i-am-not-a-travel-agent-and-have-no-experience-in-the-travel-industry,-can-i-still-use-the-self-service-apis)</li></ul>              |
```

Comment:
I see that there are several individual APIs sections below, shall we change this section to be 'Self-Service APIs 
General' or 'General Self-Service APIs'? I was looking for some API functional questions here but it is not.

Reply:
I agree with @minjikarin Nico, a name like "general 'as Minji suggested, or "overview" or "framework" could make it clear

Reply:
Thanks @minjikarin , @tsolakoua , changed to **Self-Service APIs general**